.github/workflows/tests.yml

```
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "pypy-3.7", "pypy-3.8"]
+        python-version: ["3.8", "3.9", "3.10", "pypy-3.8"]
```

```suggestion
        python-version: ["3.8", "3.9", "3.10", "pypy-3.8", "pypy-3.9"]
```
and 3.10 is on its way